kolibri/plugins/device/assets/src/views/DeviceInfoPage.vue

```diff
@@ -1,72 +1,81 @@
 <template>
 
-  <div>
-    <h1>{{ $tr('header') }}</h1>
-    <table :class="windowIsSmall ? 'mobile-table' : ''">
-      <tr>
-        <th>
-          {{ $tr('url', { count: deviceInfo.urls.length }) }}
-        </th>
-        <td :class="windowIsSmall ? 'mobile' : ''">
-          <KExternalLink
-            v-for="(url, index) in deviceInfo.urls"
-            :key="index"
-            :text="url"
-            :href="url"
-            :primary="true"
-            :openInNewTab="true"
-            appearance="basic-link"
-          />
-        </td>
-      </tr>
-      <tr>
-        <th>{{ $tr('freeDisk') }}</th>
-        <td>{{ deviceInfo.content_storage_free_space }}</td>
-      </tr>
-      <tr>
-        <th>
-          {{ $tr('kolibriVersion') }}
-        </th>
-        <td :class="windowIsSmall ? 'mobile' : ''">
-          {{ deviceInfo.version }}
-        </td>
-      </tr>
-      <tr>
-        <th>{{ coreString('deviceNameLabel') }}</th>
-        <td>
-          {{ deviceNameWithId }}
+  <AppBarPage :title="pageTitle">
+
+    <template #subNav>
+      <DeviceTopNav v-if="!hideSubNav" />
+    </template>
+
+    <KPageContainer class="device-container">
```

Comment:
For example here, I can imagine this class getting deleted as it seems to be unused in the component?